test/puppetlabs/puppetdb/command_test.clj

```diff
@@ -1366,6 +1368,27 @@
                  (get "deactivated")
                  (pt/from-string)))))))
 
+(deftest command-response-channel
+  (svc-utils/with-puppetdb-instance
+    (let [pdb (get-service svc-utils/*server* :PuppetDBServer)
+          dispatcher (get-service svc-utils/*server* :PuppetDBCommandDispatcher)
+          shared-globals (partial cli-svc/shared-globals pdb)
+          enqueue-command (partial enqueue-command dispatcher)
+          globals (shared-globals)
+          {{:keys [connection endpoint]} :command-mq} globals
+          response-mult (response-mult dispatcher)
+          response-chan (async/chan)
+          command-uuid (ks/uuid)]
+      (async/tap response-mult response-chan)
+      (enqueue-command connection endpoint
+                       (command-names :deactivate-node) 3
+                       {:certname "foo.local" :producer_timestamp (java.util.Date.)}
+                       command-uuid)
+      (is (= command-uuid
+             (async/alt!!
+               response-chan ([msg] (:id msg))
+               (async/timeout 2000) ::timeout))))))
```

Comment:
I might pull this out into a let, at first glance I thought you were comparing a channel to uuid